src/js/components/Layer/StyledLayer.js

```diff
@@ -197,6 +197,15 @@ const KEYFRAMES = {
   },
 };
 
+const getKeyframes = (animate, position, full) => {
+  if (!animate) {
+    return 'none';
```

Comment:
i dont think `none` is a valid `animation` value. I feel `animation` should not be added if `animate` is false.